docs/feature_oled_driver.md

```diff
@@ -2,7 +2,16 @@
 
 ## OLED Supported Hardware
 
-128x32 OLED modules using SSD1306 driver IC over I2C. Supported on AVR based keyboards. Possible but untested hardware includes ARM based keyboards and other sized OLED modules using SSD1306 over I2C, such as 128x64. 
+OLED modules using SSD1306 or SH1106 driver ICs, communicating over I2C.
+Tested combinations:
+
+| IC driver |   Size | Keyboard Platform | Notes                    |
+|-----------|--------|-------------------|--------------------------|
+| SSD1306   | 128x32 | AVR               | Primary support          |
+| SSD1306   | 128x32 | ARM               |                          |
```

Comment:
128x64 AVR has been tested on AVR and verified working

Reply:
Noted!